redirects.json

```diff
@@ -0,0 +1,293 @@
+  },
+  {
+    "fromPath": "/terms-of-use/",
+    "toPath": "/en/terms-of-use/",
+    "force": true
+  },
+  {
+    "fromPath": "/cookie-policy/",
+    "toPath": "/en/cookie-policy/",
+    "force": true
+  },
+  {
+    "fromPath": "/languages/",
+    "toPath": "/en/languages/",
+    "force": true
+  },
+  {
+    "fromPath": "/enterprise/",
+    "toPath": "/en/enterprise/",
+    "force": true
+  },
+  {
+    "fromPath": "/java/",
+    "toPath": "/en/developers/docs/programming-languages/java/",
+    "force": true
+  },
+  {
+    "fromPath": "/python/",
+    "toPath": "/en/developers/docs/programming-languages/python/",
+    "force": true
+  },
+  {
+    "fromPath": "/javascript/",
+    "toPath": "/en/developers/docs/programming-languages/javascript/",
+    "force": true
+  },
+  {
+    "fromPath": "/golang/",
+    "toPath": "/en/developers/docs/programming-languages/golang/",
+    "force": true
+  },
+  {
+    "fromPath": "/rust/",
+    "toPath": "/en/developers/docs/programming-languages/rust/",
+    "force": true
+  },
+  {
+    "fromPath": "/dot-net/",
+    "toPath": "/en/developers/docs/programming-languages/dot-net/",
+    "force": true
+  },
+  {
+    "fromPath": "/delphi/",
+    "toPath": "/en/developers/docs/programming-languages/delphi/",
+    "force": true
+  },
+  {
+    "fromPath": "/dart/",
+    "toPath": "/en/developers/docs/programming-languages/dart/",
+    "force": true
+  },
+  {
+    "fromPath": "/nft/",
+    "toPath": "/en/nft/"
+  },
+  {
+    "fromPath": "/nfts/",
+    "toPath": "/en/nft/"
+  },
+  {
+    "fromPath": "/dao/",
+    "toPath": "/en/dao/"
+  },
+  {
+    "fromPath": "/daos/",
+    "toPath": "/en/dao/"
+  },
+  {
+    "fromPath": "/defi/",
+    "toPath": "/en/defi/"
+  },
+  {
+    "fromPath": "/no/*",
+    "toPath": "/nb/:splat"
+  },
+  {
+    "fromPath": "/en/java/",
+    "toPath": "/en/developers/docs/programming-languages/java/"
+  },
+  {
+    "fromPath": "/en/python/",
+    "toPath": "/en/developers/docs/programming-languages/python/"
+  },
+  {
+    "fromPath": "/en/javascript/",
+    "toPath": "/en/developers/docs/programming-languages/javascript/"
+  },
+  {
+    "fromPath": "/en/golang/",
+    "toPath": "/en/developers/docs/programming-languages/golang/"
+  },
+  {
+    "fromPath": "/en/rust/",
+    "toPath": "/en/developers/docs/programming-languages/rust/"
+  },
+  {
+    "fromPath": "/en/dot-net/",
+    "toPath": "/en/developers/docs/programming-languages/dot-net/"
+  },
+  {
+    "fromPath": "/en/delphi/",
+    "toPath": "/en/developers/docs/programming-languages/delphi/"
+  },
+  {
+    "fromPath": "/en/dart/",
+    "toPath": "/en/developers/docs/programming-languages/dart/"
+  },
+  {
+    "fromPath": "/en/developers/docs/mining/",
+    "toPath": "/en/developers/docs/consensus-mechanisms/pow/mining/"
+  },
+  {
+    "fromPath": "/en/eth2/the-beacon-chain/",
+    "toPath": "/en/eth2/beacon-chain/"
+  },
+  {
+    "fromPath": "/*/beginners",
+    "toPath": "/:splat/what-is-ethereum/"
```

Comment:
These are incorrect - I think we want `"toPath": "/*/what-is-ethereum/"` - see the "splat redirects" section: https://support.gatsbyjs.com/hc/en-us/articles/1500003051241-Working-with-Redirects